EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -160,7 +166,8 @@ public function attachRefreshToken(AuthenticationSuccessEvent $event): void
                     $this->cookieSettings['secure'],
                     $this->cookieSettings['http_only'],
                     false,
-                    $this->cookieSettings['same_site']
+                    $this->cookieSettings['same_site'],
+                    $this->cookieSettings['partitioned']
```

Comment:
If you add a trailing comma at the end of this line,  future changes that add a line will only need to edit one line and not two.

Hardly a necessary change but perhaps something worth considering.

Reply:
Fair point! 

I hadn't done this because I wasn't sure which PHP version added support for trailing commas and didn't want it to break on older versions, but I just looked it up and it was added in 7.3. 

Seeing as this package supports `>= 7.4`, I just added the trailing comma :)